README.md

```diff
@@ -1,3 +1,28 @@
 # buildbot-macports-custom-views
 
 Custom views plugin for MacPorts Buildbot.
+
+## Getting Started
+
+### Set up buildbot environment
+
+Set up a buildbot environment: http://docs.buildbot.net/latest/developer/quickstart.html
```

Comment:
Just out of curiosity: does one really need to install buildbot in "full developer mode" as described under http://docs.buildbot.net/latest/developer/quickstart.html#create-a-buildbot-python-environment for the sake of using (or maybe slightly modifying) this plugin?

Reply:
> Just out of curiosity: does one really need to install buildbot in "full developer mode" as described under http://docs.buildbot.net/latest/developer/quickstart.html#create-a-buildbot-python-environment for the sake of using (or maybe slightly modifying) this plugin?

Apparently, installing buildbot in developer mode isn't needed.

Reply:
A better option is to use `python3 -m venv .venv`. I think this should work on mac and linux

Reply:
As a replacement for https://github.com/buildbot/buildbot/blob/master/Makefile#L82 ?

Reply:
Yes, but it doesn't work with python2 (anyways buildbot doesn't use python2 now)